src/Console/Migrate.php

```diff
@@ -123,8 +265,39 @@ protected function resolveBuilders(): void
         $this->target = $this->resolveBuilder($this->target());
     }
 
+    protected function resolveOptions(): void
+    {
+        $this->table_names                  = $this->tableNames();
+        $this->exclude_tables               = $this->excludeTables();
+        $this->retrive_tables_from_target   = false;
+        $this->truncate_tables              = false;
+        $this->drop_target_tables           = false;
```

Comment:
The initial state of a parameter must be defined in the class, not in the called method.